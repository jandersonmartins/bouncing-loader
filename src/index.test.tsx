import React from 'react'
import '@testing-library/jest-dom/extend-expect'
import 'jest-styled-components'
import { render } from '@testing-library/react'
import { BouncingLoader } from '.'

test('should render correctly', () => {
  const { container } = render(<BouncingLoader color='#3700B3' />)

  expect(container.firstChild).toMatchSnapshot()
})
