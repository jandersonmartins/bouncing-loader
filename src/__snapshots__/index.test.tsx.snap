// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
.c0 {
  position: absolute;
  left: 0;
  top: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 100vw;
  height: 100vh;
  font-size: 1.6rem;
}

.c1 {
  width: 1.5rem;
  height: 1.5rem;
  border-radius: 50%;
  background-color: #3700B3;
  -webkit-animation: bounce 0.4s infinite alternate;
  animation: bounce 0.4s infinite alternate;
}

.c1:first-child {
  -webkit-animation-delay: 0.6s;
  animation-delay: 0.6s;
}

.c1:last-child {
  -webkit-animation-delay: 0.2s;
  animation-delay: 0.2s;
}

.c2 {
  width: 1.5rem;
  height: 1.5rem;
  border-radius: 50%;
  background-color: #3700B3;
  margin: 0 .5rem;
  -webkit-animation: bounce 0.4s infinite alternate;
  animation: bounce 0.4s infinite alternate;
}

.c2:first-child {
  -webkit-animation-delay: 0.6s;
  animation-delay: 0.6s;
}

.c2:last-child {
  -webkit-animation-delay: 0.2s;
  animation-delay: 0.2s;
}

<div
  class="c0"
>
  <div
    class="c1"
    color="#3700B3"
  />
  <div
    class="c2"
    color="#3700B3"
  />
  <div
    class="c1"
    color="#3700B3"
  />
</div>
`;
